conf/modules/nav_fish.xml

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="nav_fish" dir="nav">
+  <doc>
+    <description>
+      Bio-inspired swarm navigation
```

Comment:
If available, a link to HAL or similar to the algorithm description?

Reply:
done